### PR TITLE
Added YouTube video to contributing file with the img (#7289)

### LINKED ISSUE #7289: [DOCS] Add YouTube video to contributing file
### Description

Add this video with thumbnail to the top o the contributing guide https://youtu.be/dfeSpGd8leU

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by luckyklyist**
## Fixes Issue 

fixes #7289 

Added a YouTube video thumbnail with a link to the Contributing Guide video for LinkFree.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot from 2023-06-01 14-25-27

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7304"></a>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Contributing Guide
 
+[![LinkFree Contributing Guide](https://github-production-user-asset-6210df.s3.amazonaws.com/35479077/242535587-d33e07a3-5732-465e-955a-7d7bd504b99b.jpg)](https://youtu.be/dfeSpGd8leU)
+
+## Issues & Pull Requests (not for Profiles)
```
Comment (eddiejaoude) [inline CONTRIBUTING.md:5]: This is a duplicate title

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Contributing Guide
 
+[![LinkFree Contributing Guide](https://github-production-user-asset-6210df.s3.amazonaws.com/35479077/242535587-d33e07a3-5732-465e-955a-7d7bd504b99b.jpg)](https://youtu.be/dfeSpGd8leU)
```
Comment (eddiejaoude) [inline CONTRIBUTING.md:3]: It would be good to have some text below to show this is a video

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you, I left some inline comments

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you, I left some inline comments

**Comment by luckyklyist** [comment]
image
I have added the text below to describe the image: #7289.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
 # Contributing Guide
 
+[![LinkFree Contributing Guide](https://user-images.githubusercontent.com/82668196/242340741-a0124ead-97b6-488f-9271-10f2b0e1f577.jpg)](https://youtu.be/dfeSpGd8leU)
+You can watch the video that Eddie created as a contributing guide. Click on the image to watch the video.
+ 
 ## Issues & Pull Requests (not for Profiles)
 
+
```
Comment (eddiejaoude) [inline CONTRIBUTING.md:8]:
```suggestion
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Contributing Guide
 
+[![LinkFree Contributing Guide](https://user-images.githubusercontent.com/82668196/242340741-a0124ead-97b6-488f-9271-10f2b0e1f577.jpg)](https://youtu.be/dfeSpGd8leU)
+You can watch the video that Eddie created as a contributing guide. Click on the image to watch the video.
```
Comment (eddiejaoude) [inline CONTRIBUTING.md:4]:
```suggestion

You can watch the video that Eddie created as a contributing guide. Click on the image to watch the video.
```

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**